scripts/extensions-emulator-tests/tests.ts

```diff
@@ -0,0 +1,62 @@
+import { expect } from "chai";
+import * as fs from "fs";
+import * as path from "path";
+import * as subprocess from "child_process";
+
+import { FrameworkOptions, TriggerEndToEndTest } from "../integration-helpers/framework";
+
+const EXTENSION_ROOT = path.dirname(__filename) + "/greet-the-world";
+
+const FIREBASE_PROJECT = process.env.FBTOOLS_TARGET_PROJECT || "";
+const FIREBASE_PROJECT_ZONE = "us-east1";
+const TEST_CONFIG_FILE = "test-firebase.json";
+const TEST_FUNCTION_NAME = "greetTheWorld";
+
+/*
+ * Various delays that are needed because this test spawns
+ * parallel emulator subprocesses.
+ */
+const TEST_SETUP_TIMEOUT = 60000;
+const EMULATORS_SHUTDOWN_DELAY_MS = 5000;
+
+function readConfig(): FrameworkOptions {
+  const filename = path.join(EXTENSION_ROOT, "test-firebase.json");
+  const data = fs.readFileSync(filename, "utf8");
+  return JSON.parse(data);
+}
+
+describe("extension emulator", () => {
+  let test: TriggerEndToEndTest;
+
+  before(async function() {
+    this.timeout(TEST_SETUP_TIMEOUT); // eslint-disable-line no-invalid-this
+
+    expect(FIREBASE_PROJECT).to.exist.and.not.be.empty;
+
+    // TODO(joehan): Delete the --open-sesame call when extdev flag is removed.
+    const p = subprocess.spawnSync("firebase", ["--open-sesame", "extdev"], { cwd: __dirname });
+    console.log("open-sesame output:", p.stdout.toString());
```

Comment:
Clean up console.log

Reply:
This one is actually intentional. `spawnSync` isn't printing the normal output (because I think other things have taken over the pipes), so I'm manually printing it here.